wwwroot/assets/scripts/display.js

```diff
@@ -0,0 +1,117 @@
+const $ = document.querySelector.bind(document);
+import * as g from './global';
+import {formatNumber} from './utils';
+
+export const fillMapAndLegend = () => {
+  updateTitle();
+  updateMapColors();
+  displayLegend();
+};
+
+export const updateMapColors = () => {
+  // assign CSS class each to svg element if valid value
+  for (let svgCountry of g.svgCountries) {
+
+    let svgCountryCode = svgCountry.getAttribute('data-id');
+
+    let countryValue = g.countries.getValue(svgCountryCode, g.activeIndicator, g.activeYear);
+
+    let classCountry = (countryValue === '')
```

Comment:
as code standard, It's a bit weird to see it in 3 lines; And the parenthesis aren't needed.
```javascript
let classCountry = country === ''? 'noData' : getClassName(countryValue, g.activeIndicator, g.activeYear);
```